apps/admin-gui/src/assets/i18n/en.json

```diff
@@ -2192,6 +2192,11 @@
       "GROUP_RELATION": "User's membership in all groups",
       "SUCCESS_NOTIFICATION": "User was successfully anonymized"
     },
+    "DELETE_USER": {
+      "VO_RELATION": "User's membership in all organizations",
+      "GROUP_RELATION": "User's membership in all groups",
```

Comment:
Since this action is quite destructive are there other relations worth mentioning ? I am not sure but we can discuss.